data/map planets.txt

```diff
@@ -2649,6 +2706,15 @@ planet Memory
 		threshold 4000
 		fleet "Small Deep Security" 16
 
+planet "Mendez Station"
```

Comment:
I'll wait for this to be rewritten and I'll submit a review later.

Reply:
Rewrite done. Looking forward to your feedback.